javacutil/src/org/checkerframework/javacutil/ElementUtils.java

```diff
@@ -360,14 +360,25 @@ public static boolean hasReceiver(Element element) {
             // For each direct supertype of the current type element, if it
             // hasn't already been visited, push it onto the stack and
             // add it to our superelems set.
-            TypeMirror supertypecls = current.getSuperclass();
-            if (supertypecls.getKind() != TypeKind.NONE) {
+            TypeMirror supertypecls;
+            try {
+                supertypecls = current.getSuperclass();
```

Comment:
There are 18 other calls to getSuperClass in the Checker Framework.  Could all of them also issue a CompletionFailure or just this location?